tests/zexperimental/test_mcmc.py

```diff
@@ -340,7 +339,7 @@ def test_NUTS_regression_warmup(target: cuqi.density.Density):
     cuqi.experimental.mcmc.SamplerNew,
     cuqi.experimental.mcmc.ProposalBasedSamplerNew,
     cuqi.experimental.mcmc.MHNew,
-    cuqi.experimental.mcmc.pCNNew,
+    cuqi.experimental.mcmc.PCNNew,
```

Comment:
Could you try adding the PCNNew to the list that checks for checkpointing? Any issues still?

Reply:
Unfortunately, PCNNew still doesn't pass checkpointing. Maybe we should work on it later with issue #386 ?

Reply:
Ok. Too bad :) Let us postpone fixing it to #386 yes.